Packages/com.chisel.components/Chisel/Components/API.public/BrushMeshAssetFactory/Cylinder/BrushFactory.Cylinder.cs

```diff
@@ -19,18 +19,40 @@ namespace Chisel.Components
     // TODO: rename
     public sealed partial class BrushMeshAssetFactory
     {
-        public static bool GenerateCylinderAsset(CSGBrushMeshAsset brushMeshAsset, CSGCylinderDefinition bottom, float topHeight, float rotation, int sides, CSGSurfaceAsset[] surfaceAssets, SurfaceDescription[] surfaceDescriptions)
+        public static bool GenerateCylinderAsset(CSGBrushMeshAsset brushMeshAsset, CSGCylinderDefinition definition)
```

Comment:
All the data that defines a cylinder is in it's own struct now, to make it consistent with all other generators